stock_analysis/report/stock_analysis.py

```diff
@@ -10,12 +9,13 @@ class StockAnalysis(models.Model):
     _name = 'stock.analysis'
     _auto = False
     _rec_name = 'product_id'
+    _description = 'Stock analysis view'
 
     product_id = fields.Many2one(
         'product.product', string='Product', readonly=True)
     location_id = fields.Many2one(
         'stock.location', string='Location', readonly=True)
-    qty = fields.Float(string='Quantity', readonly=True)
+    quantity = fields.Float(string='Quantity', readonly=True)
```

Comment:
Even in a view, I think you should add decimal precision

Reply:
This is based on `quantity` fields of `stock.quant`, that does not have precision